src/contacts/model/Record.java

```diff
@@ -0,0 +1,69 @@
+package contacts.model;
+
+import contacts.Constants;
+
+import java.time.LocalDateTime;
+
+public abstract class Record {
+
+    protected String name;
+    protected String phoneNumber;
+    protected LocalDateTime creationDate;
+    protected LocalDateTime lastEditeDate;
+
+    private boolean isPerson;
```

Comment:
What is it for?

Reply:
This is to separate the Record into a Person and an Organization?